src/main/java/Event.java

```diff
@@ -0,0 +1,45 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Event class is a type of task.
+ */
+public class Event extends Task{
+    LocalDate date;
+
+    /**
+     * The constructor for the Event class.
+     * @param description The description of the event.
+     * @param date The date for the event.
+     */
+    public Event(String description, LocalDate date) {
+        super(description);
+        this.date = date;
+        totalTasks++;
```

Comment:
This can be moved to the parent class's constructor as you are increasing number of tasks (not number of events)